### ISSUE TITLE
Please comment on VirtualSpace's multiple monitor support.

### ISSUE THREAD

**Issue by mdear-lsn** [newlooper/VirtualSpace#68]
Task View is missing the extensive rules-based assignment feature your app has, which is why I'm so interested in making this work.  I don't know of any other alternative that will allow me to place some of my browser windows on one virtual display and other windows from the same browser on another virtual display.  This is a key feature for me.    This means, as I understand it, that if I reboot my laptop and my browser (Brave in my case) launches many windows, they will be distributed to the appropriate desktop and virtual monitor as previously configured.  It is very valuable to me to automatically and consistently associate all browser windows to certain online news sources to a "Personal" VirtualSpace and all browser windows accessing certain technical websites being associated with a "Work" VirtualSpace.   Same browser, many windows, many purposes, many VirtualSpaces.

I have three screens, one inbuilt (laptop) and two external.  When I select a virtual desktop, does your app behave like Windows 10 Task View, in which each virtual desktop is replicated across all available physical screens?  When I launch your app, unlike Task View, I only see the VirtualSpace window on one physical screen.

Also, what happens when I unplug my laptop and are reduced to a single inbuilt screen?  And what happens when I plug into my home office and regain my three screens again?  And what happens when I plug into my work office and a different set of monitors appears?

In other words, when I select workspace 1, I have three virtual displays that I can sent apps to.   Activating wordspace 1 sends those three virtual displays to three physical displays.  When I select workspace 2, I have three additional virtual displays that I can send apps to.   If the assigned display isn't available, I would expect to see those apps on the available screen (fallback).  I would also expect it to be possible to send (replicate) apps to one or more screens (My office setup consists of two different monitors than the ones I have at home, yet I want to configure similar placement between the two locations despite these differences to ensure consistency).

**Comment by mdear-lsn** [comment]
My experience so far is that VS works well with multiple combinations of multiple monitors and scales well from single (inbuilt laptop) monitor to one or more external monitors.   I do not populate rules that direct to a particular monitor (because I'm still not sure how that works with using multiple combinations of monitors).

**Comment by mdear** [comment]
This issue may be closed.
